lib/miq_automation_engine/models/miq_ae_domain.rb

```diff
@@ -45,6 +49,45 @@ def available_version
     version_field.try(:fetch_path, 'field', 'default_value')
   end
 
+  def self.import_git_repo(domain_name, git_repo, branch = nil, tag = nil)
```

Comment:
Is it possible to get both a branch and a tag?  If not, I think it's confusing to have both as parameters.  Wondering if it's cleaner to have `ref` and `ref_type` instead